lib/tasks/foreman_fog_proxmox_tasks.rake

```diff
@@ -15,7 +15,7 @@ namespace :test do
 end
 
 namespace :foreman_fog_proxmox do
-  task :rubocop do
+  task :rubocop => :environment do
```

Comment:
environment is declared in function arg but is not used in the body function

Reply:
This is a finding from rubocop. It shouldn't break anything.